profile/prune.go

```diff
@@ -22,6 +22,37 @@ import (
 	"strings"
 )
 
+var (
+	reservedNames = []string{"(anonymous namespace)", "operator()"}
+	bracketRx     = regexp.MustCompile("(" + strings.Replace(strings.Replace(
```

Comment:
is this `strings.Replace` dance really just [`regexp.QuoteMeta`](https://godoc.org/regexp#QuoteMeta)?